sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/persons/PersonDirectorySteps.java

```diff
@@ -281,10 +288,29 @@ public PersonDirectorySteps(
           webDriverHelpers.selectFromCombobox(PRESENT_CONDITION_FILTER_COMBOBOX, "Alive");
         });
 
+    When(
+        "I check if Date of dead for specified case is correct",
+        () -> {
+          String date = webDriverHelpers.getValueFromWebElement(DATE_OF_DEATH_INPUT);
+          LocalDate deadDate = LocalDate.now().minusDays(1);
+          softly.assertEquals(DATE_FORMATTER.format(deadDate), date);
+          softly.assertAll();
+        });
+
+    When(
+        "I check if Cause of death is ([^\"]*)",
+        (String causeOfDeath) -> {
+          String deathCause = webDriverHelpers.getValueFromCombobox(CASE_OF_DEATH_COMBOBOX);
+          softly.assertEquals(deathCause, causeOfDeath, "Cause of death is not equal");
+          softly.assertAll();
+        });
+
     When(
         "I click on first person in person directory",
         () -> {
           webDriverHelpers.clickOnWebElementBySelector(By.cssSelector("[role='gridcell'] a"));
+          webDriverHelpers.waitForPageLoaded();
```

Comment:
remove wait

Reply:
that is not my method i hope that remove will not fail other test